src/explore-education-statistics-admin/src/services/releaseChartFileService.ts

```diff
@@ -28,36 +30,33 @@ function mapFile(file: FileInfo): ChartFile {
 }
 
 const releaseChartFileService = {
-  async getChartFiles(releaseId: string): Promise<ChartFile[]> {
-    const response = await client.get<FileInfo[]>(
-      `/release/${releaseId}/chart`,
-    );
-
-    return response.map(mapFile);
-  },
-
   async uploadChartFile(
     releaseId: string,
+    id: string,
     request: UploadChartFileRequest,
   ): Promise<ChartFile> {
     const data = new FormData();
     data.append('file', request.file);
+    let file;
 
-    const file = await client.post<FileInfo>(
-      `/release/${releaseId}/chart`,
-      data,
-    );
-
+    if (id !== '') {
```

Comment:
Would make this check `if (id)` to leverage coercion. This protects us from other empty values potentially passing this test by accident.